apis/nucleus/src/components/Header.jsx

```diff
@@ -83,7 +83,13 @@ function Header({ layout, sn, anchorEl, hovering, focusHandler, titleStyles = {}
         <Grid container wrap="nowrap" direction="column">
           {showTitle && (
             <Tooltip title={layout.title}>
-              <Typography variant="h6" noWrap className={CellTitle.className} style={titleStyles.main}>
+              <Typography
+                id={`${layout.qInfo.qId}_title`}
```

Comment:
This might not be unique within a html page, there is a `currentId` in Cell you can use. (same below for `${layout.qInfo.qId}_content` and the `${layout?.qInfo.qId}_type`)